src/Microsoft.AspNetCore.Authentication.OpenIdConnect/OpenIdConnectOptions.cs

```diff
@@ -167,15 +167,16 @@ public new OpenIdConnectEvents Events
 
         /// <summary>
         /// The request path within the application's base path where the user agent will be returned after sign out from the identity provider.
+        /// See post_logout_redirect_uri.
```

Comment:
Is this comment supposed to be helpful? 'cause it's... quite succinct :trollface: